main.py

```diff
@@ -0,0 +1,16 @@
+"""Prepare image CVE data for publishing publicly."""
+
+import os
+
+import pandas as pd
+
+from filter_data import filter_df
+
+# Details about bucket from which to download images CVE data
+BUCKET = os.environ.get("BUCKET")
+
+if __name__ == "__main__":
+    df = pd.read_csv(f"gs://{BUCKET}/latest.csv")
```

Comment:
So this is available at a public http location - wondering if we would rather do that?

Reply:
That seems easier indeed! Let me use a public URL instead.

Reply:
Fixed with commit [5353930](https://github.com/chainguard-dev/image-comparison/pull/11/commits/53539301e8fbb4d23cf455c7532b7afd52429579)